versioned_docs/version-2.0/tutorials/Administrators/manage_spend.mdx

```diff
@@ -0,0 +1,157 @@
+:::
+
+Choosing the right percentage of runs to sample depends on your use case. We arbitrarily picked 10% of runs here, but will
+leave it to the user to find the right value that balances collecting rare events and cost constraints.
+
+### See results after 7 days
+While the total amount of traces per day stayed the same, the extended data retention traces as expected was cut heavily.
+
+![](./static/p1endresultgraph.png)
+
+This translates to the invoice, where we've only added an extra $800 in spend in the last week, meaning we've cut weekly
+costs by nearly 80%! 
+
+![](./static/p1endresultinvoice.png)
+
+## Optimization 2: limit usage
+
+In the previous section, we managed data retention settings to *optimize existing spend*. In this section, we will
+use usage limits to *prevent future overspend*. 
+
+LangSmith has two usage limits: total traces and extended retention traces. These correspond to the two metrics we've 
+been tracking on our usage graph. We can use these in tandem to have granular control over spend. 
+
+To set limits, we navigate to `Settings` -> `Usage and Billing` -> `Usage Limits`. We see our two limits appear, along
+with a cost estimate at the bottom:
+
+![](./static/p2usagelimitsempty.png)
+
+### Picking a good all traces limit 
+Picking the right "all traces" limit depends on the expected load of traces that you will send to LangSmith. For example,
+our gen AI application is called between 1.2-1.5 times per second, and each API request has a trace associated with it,
+meaning we log around 100,000-130,000 traces per day. We expect to double in size in the near future, so we'd ideally like to set our limit
+to `2 * 130,000 * 30 = 7,800,000` traces per month.
+
+![](./static/p2alllimitonly.png)
+
+:::note
+When set without the all traces limit, the maximum cost estimator assumes that all traces are using extended data retention.
+:::
+
+### Cutting maximum spend with an extended data retention limit
+If we are not a big enterprise, we may shutter at the ~$40k per month bill at Plus plan prices.
+
+We saw from Optimization 1 that the easiest way to cut cost was through data retention. The same can be said for limits. 
+If we only want to keep ~10% of traces to be around more than 14 days, we can set a limit on the maximum high retention
+traces we can keep. That would be `.10 * 7,800,000 = 780,000`. 
+
+![](./static/p2bothlimits.png)
+
+As we can see, the maximum cost is cut from ~40k per month to ~7.5k per month. This lets us be confident that new 
+users on the platform will not accidentally cause cost to balloon.
+
+:::note
+The extended data retention limit can cause features other than traces to stop working once reached. If you plan to 
+use this feature, please read more about its functionality here.
+:::
+
+## Summary
+In this tutorial, we learned how to:
+
+1. Cut down our existing costs with data retention policies
+2. Prevent future overspend with usage limits
```

Comment:
Should we add a section (3) around cleaning up run rules that have high sampling rates?